packages/api/app/Models/Permission.js

```diff
@@ -23,6 +24,37 @@ class Permission extends Model {
 		this.addTrait('Params');
 	}
 
+	static async create(payload) {
+		let modelInstance;
+		const permission = await this.findBy('permission', payload.permission);
+		if (!permission) {
+			modelInstance = new Permission();
+			modelInstance.fill(payload);
+			await modelInstance.save();
```

Comment:
Creio que assim funcione...
```suggestion
			await super(payload);
```

Reply:
Não funciona 👎

Reply:
E `super.create()`?